dcrdex/go.mod

```diff
@@ -1,8 +1,8 @@
 module decred.org/release/dcrdex
 
-go 1.15
+go 1.16
 
 require (
-	decred.org/dcrdex v0.2.0
-	decred.org/dcrdex-assets v0.2.0
+	decred.org/dcrdex v0.0.0-20211217203335-620a0258b8cd
+	decred.org/dcrdex-assets v0.4.0-rc1
```

Comment:
You can stick with this for `dcrdex-assets`, or go with [3616e97a72c7ce8](https://github.com/decred/dcrdex-assets/commit/3616e97a72c7ce836b2e51d58dc08f2c4cda211d), which is one commit past rc1 with a minor charts display fix.